src/app/config-screen/config-screen.component.html

```diff
@@ -1,84 +1,82 @@
+                   (ngModelChange)="setGitHubConfigValue('watchList', $event)"/>
           </div>
         </div>
       </div>
     </div>
 
-    <div class="col-md-6">
-    <div class="panel panel-primary">
-      <div class="panel-heading">
-        <h3 class="panel-title">
+    <div class="col-6">
+      <div class="card">
+        <div class="card-header">
           <i class="icon fa fa-windows"></i>
           VSTS Config
-          <span *ngIf="configService.vsts.isConfigured()" class="label label-success">Valid Input</span>
-          <span *ngIf="!configService.vsts.isConfigured()" class="label label-danger">Invalid Input</span>
-        </h3>
-      </div>
-      <div class="panel-body">
-        <div class="form-group">
-          <label>Team Name</label>
-          <select class="form-control input-lg"
-            [ngModel]="getVstsConfigValue('team')" (ngModelChange)="setVstsConfigValue('team', $event)">
-            <option *ngFor="let team of teams" [ngValue]="team">{{team}}</option>
-          </select>
-        </div>
-        <div class="form-group">
-          <label>User Name</label>
-          <input type="text" class="form-control input-lg" placeholder="first.last@blackbaud.me"
-            [ngModel]="getVstsConfigValue('username')" (ngModelChange)="setVstsConfigValue('username', $event)" />
+          <span *ngIf="configService.vsts.isConfigured()" class="badge badge-success">Valid Input</span>
+          <span *ngIf="!configService.vsts.isConfigured()" class="badge badge-danger">Invalid Input</span>
         </div>
-        <div class="form-group">
-          <label>Token</label>
-          <input type="text" class="form-control input-lg"
-            [ngModel]="getVstsConfigValue('token')" (ngModelChange)="setVstsConfigValue('token', $event)" />
+        <div class="card-body">
+          <div class="form-group">
+            <label>Team Name</label>
+            <select class="form-control input-lg"
+                    [ngModel]="getVstsConfigValue('team')" (ngModelChange)="setVstsConfigValue('team', $event)">
+              <option *ngFor="let team of teams" [ngValue]="team">{{team}}</option>
+            </select>
+          </div>
+          <div class="form-group">
+            <label>User Name</label>
+            <input type="text" class="form-control input-lg" placeholder="first.last@blackbaud.me"
+                   [ngModel]="getVstsConfigValue('username')" (ngModelChange)="setVstsConfigValue('username', $event)"/>
+          </div>
+          <div class="form-group">
+            <label>Token</label>
+            <input type="text" class="form-control input-lg"
+                   [ngModel]="getVstsConfigValue('token')" (ngModelChange)="setVstsConfigValue('token', $event)"/>
+          </div>
         </div>
       </div>
     </div>
   </div>
-  </div>
 
 
-  <div class="col-md-10 col-md-offset-1 row text-center">
+  <div class="col-1 offset-5 row text-center">
```

Comment:
Based on the picture attached, it doesn't seem like the save button really needed more centering. In fact, the updated picture makes it look like the save button is skewed toward the left side of the screen.

Reply:
yea, i messed that up. pushed a fix. upgrading ended up moving the button. but i centered it via the new bootstrap classes.